consensus/qbft/src/test/java/org/hyperledger/besu/consensus/qbft/statemachine/QbftRoundTest.java

```diff
@@ -196,90 +194,6 @@ public void onReceptionOfValidProposalSendsAPrepareToNetworkPeers() {
     verify(transmitter, never()).multicastCommit(any(), any(), any());
   }
 
-  @Test
-  public void sendsAProposalAndPrepareWhenSendProposalRequested() {
```

Comment:
Won't block the PR on this, but wondering if these deleted tests make sense to be ported over to QbftBlockHeightManagerTest?

IIUC, the logic that was in QbftRound.createAndSendProposalMessage has been folded into QbftBlockHeightManager

Reply:
Well, I have separated the _createAndSendProposalMessage_ in two, _createBlock_  and _sendProposalMessage_ , but this last one only contained a call to _updateStateWithProposalAndTransmit_, so it was redundant to have it, has @matthew1001 noted. And then I noticed that _createAndSendProposalMessage_ was only called from tests, thus it was removed along with the tests that called.

It may make sense to do some tests around _createBlock_  + _updateStateWithProposalAndTransmit_ yes, I can take a look.

Cheers.